net.runelite.RuneLite.json

```diff
@@ -17,7 +17,7 @@
         "--env=JAVA_HOME=/app/jre",
         "--talk-name=org.freedesktop.Notifications",
         "--filesystem=xdg-run/app/com.discordapp.Discord:create",
-        "--filesystem=~/.runelite:create",
+        "--persist=.runelite",
         "--persist=jagexcache"
```

Comment:
this is no longer written here but instead in .runelite/jagexcache, so it can be removed

Reply:
Done, thanks! :-)